lib/kontrast/configuration.rb

```diff
@@ -71,11 +71,12 @@ def validate
             end
         end
 
-        def pages(width)
+        def pages(width, url_params = {})
```

Comment:
Just realized that I blindly used keyword params in `Kontrast::PageTest#initialize`, which makes the gem only work on 2.x+. Given that even rails is dropping 1.9 support, I think it's fine for us to not support it, but we should maybe talk about it, and regardless of the outcome, updated the README.

That being said we should either use the default value pattern everywhere or keyword arguments everywhere.